client/src/components/Reviews/components/ReviewList.test.js

```diff
@@ -0,0 +1,23 @@
+import { render, fireEvent, waitFor, screen } from '@testing-library/react';
+import ReviewList from 'Reviews/components/ReviewList.jsx';
+
+describe('test', () => {
+  test('Ratings & Reviews', async () => {
```

Comment:
It's helpful if the test statement describes what is being tested in that test. So in this case it might be something like `"the 'More reviews' button should not be disabled after click"`.